.github/workflows/check-release.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  # For all pushes run the tests
+  push:
+  # Also trigger on pull_request, as well as pull_request_target
+  pull_request:
+  pull_request_target:
+  # Run nightly tests at 04:00 UTC
+  schedule:
+    - cron: "0 4 * * *"
+
+name: docker
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    container:
+      image: python:latest
+    env:
+      GITHUB_PAT: ${{ secrets.INSTALL_GITHUB }}
+    # If the PR is coming from a fork (pull_request_target), ensure it is opened
+    # by "dependabot[bot]". Otherwise, clone it normally.
+      CC_API_ORGANISATION: ${{ secrets.CC_API_ORGANISATION }}
+      CC_API_KEY: ${{ secrets.CC_API_KEY }}
+    if: |
+        (github.event_name == 'pull_request_target' && github.actor == 'dependabot[bot]') ||
+        (github.event_name != 'pull_request_target' && github.actor != 'dependabot[bot]')
+    steps:
+      - name: Checkout
+        if: ${{ github.event_name != 'pull_request_target' }}
+        uses: actions/checkout@v4
+
+      - name: Checkout PR
+        if: ${{ github.event_name == 'pull_request_target' }}
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Setup virtualenv
+        run: |
+          python -V  # Print out python version for debugging
+          pip install virtualenv pytest
+          virtualenv venv
+          . venv/bin/activate
+
+      - name: Install dependencies
+        run: pip install coverage pytest pandas .
```

Comment:
What does the dot at the end of the line stand for?

Reply:
The main goal here is to install the pyledger and cashctrl_api packages that should be installed from the git.

Let`s go through the steps:
1. pip install coverage pytest pandas (without "." for now)
2. then we have python setup.py develop. In this scenario, I receive such an error(screenshot linked)

The "." symbol simply installs all "install_requires" dependencies. after this error vanishes
<img width="956" alt="Screenshot 2024-05-01 at 11 01 47" src="https://github.com/macxred/cashctrl_ledger/assets/90412709/ae4ccede-4217-4e5b-a8bc-a4ae6afa0bf5">

Reply:
Thanks. I just learned something :)